R/as_flex_table.R

```diff
@@ -230,7 +230,17 @@ table_styling_to_flextable_calls <- function(x, ...) {
     tidyr::nest(location_ids = c("row_numbers", "column_id")) %>%
     dplyr::mutate(
       row_numbers = map(.data$location_ids, ~ getElement(.x, "row_numbers") |> unique()),
-      column_id = map(.data$location_ids, ~ getElement(.x, "column_id") |> unique())
+      column_id =
+        pmap(
+          list(.data$location_ids,
+               .data$tab_location,
+               .data$row_numbers),
+          \(location_ids, tab_location, row_numbers) {
+            col_ids <- getElement(location_ids, "column_id") |> unique()
+            if (tab_location == "body") col_ids <- rep(col_ids, length(row_numbers)) # styler: off
+            sort(col_ids)
```

Comment:
Hi @Melkiades , I was doing some final checks, and saw that we were not placing the footnote correctly. I added the `sort()` to get it right which changed the footnote placement from this
<img width="733" alt="image" src="https://github.com/user-attachments/assets/85628cec-2818-4c58-91cf-a65448675dde">

to

<img width="676" alt="image" src="https://github.com/user-attachments/assets/db105a29-de55-4f17-a50c-9c6348eefb24">

Is there a way to test that the placement of the footnote markers is where we expect in a flextable? 

Thanks!

Reply:
Thank you!!